modules/acs-architecture-interaction-between-services.adoc

```diff
@@ -8,7 +8,7 @@
 This section explains how {product-title-short} services interact with each other.
 
 
-.{product-title-short} with Scanner V4
+.{product-title-short} with Scanner V4 (Technology Preview)
```

Comment:
This has nothing to do with your updates, but it looks like some of the icons in the table below might not be showing properly? Just something for a followup PR, if a fix is necessary

Reply:
We have a [ticket to investigate](https://issues.redhat.com/browse/ROX-23216) per an earlier peer review comment - so I'll look at it soon.